app/exports/routes.py

```
@@ -0,0 +1,71 @@
+        return "Interner Serverfehler.", 500
+
+    return send_file(
+        io.BytesIO(ical_string.encode('utf-8')),
+        as_attachment=True,
+        download_name='wochenplan.ics',
+        mimetype='text/calendar'
+    )
+
+@exports_bp.route('/download-pdf')
+def download_pdf():
+    tasks = get_task_manager().list_tasks()
```

Ruff (0.12.2)

Undefined name `get_task_manager`

(F821)